stock_reception_screen/models/stock_reception_screen.py

```diff
@@ -609,21 +615,23 @@ def process_set_expiry_date(self):
             self.warn_notification = _("You have to set an expiry date.")
             return
         if (
-            self.current_move_line_lot_id.life_date
+            self.current_move_line_lot_id.expiration_date
             and self.current_move_line_lot_life_date
-            < self.current_move_line_lot_id.life_date
+            < self.current_move_line_lot_id.expiration_date
         ):
             lang = self.env["res.lang"]._lang_get(self.env.user.lang)
-            previous_life_date_str = self.current_move_line_lot_id.life_date.strftime(
-                lang.date_format
+            previous_life_date_str = (
+                self.current_move_line_lot_id.expiration_date.strftime(lang.date_format)
             )
             self.warn_notification = _(
                 "You cannot set a date prior to previous one ({})".format(
                     previous_life_date_str
                 )
             )
             return
-        self.current_move_line_lot_id.life_date = self.current_move_line_lot_life_date
+        self.current_move_line_lot_id.expiration_date = (
+            self.current_move_line_lot_life_date
```

Comment:
I would rename all `life_date` terms to `expiration_date` to be inline with the standard (this implies a pre-migration script to rename the column).

Reply:
Hi @sebalix , I'll update the PR